client/extensions/zoninator/state/locks/actions.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	ZONINATOR_REQUEST_LOCK,
+	ZONINATOR_REQUEST_LOCK_ERROR,
+	ZONINATOR_UPDATE_LOCK,
+} from '../action-types';
+
+/**
+ * Returns an action object to indicate that a zone lock should be updated
+ *
+ * @param  {Number}  siteId          Site ID
+ * @param  {Number}  zoneId          Zone ID
+ * @param  {Date}    expires         Expiration date
+ * @param  {Number}  maxLockPeriod   Maximum number of seconds to extend the lock to
+ * @param  {Boolean} reset           Set to true to reset a lock
+ * @return {Action}                  Action object
+ */
+export const updateLock = ( siteId, zoneId, expires, maxLockPeriod, reset ) => ( {
```

Comment:
5 args is quite a few. What about `( siteId, zoneId, { expires, maxLockPeriod, reset } )`?

Alternatively, you could define a separate `resetLock` function, especially if that happens to have a different signature (e.g. does not require `expires` and/or `maxLockPeriod` (?)). Disclaimer: I don't really know how reset works.